{
  initial_frequency: 0,

  led_pin: 5,
}